app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
Code Quality Issue: The use of a class variable to store instances of `Person` might not be the best approach. It ties the `Person` class to the `create_person_list` function and makes it less reusable. It would be better to manage the collection of `Person` instances outside of the `Person` class.